avocado/core/test.py

```diff
@@ -119,6 +183,9 @@ def record_and_warn(*args, **kwargs):
         original_log_warn = self.log.warning
         self.__log_warn_used = False
         self.log.warn = self.log.warning = record_and_warn
+        if tag is not None:
+            self.log.warn("The 'tag' argument is not supported and will be "
```

Comment:
This should be `log.info()` to avoid UI having WARN for every test that otherwise would be PASS:
~~~
$ avocado run passtest failtest -m examples/mux-selftest.yaml 
JOB ID     : de316e0816d6cf71d21387a82eba2b599b1823b9
JOB LOG    : /home/apahim/avocado/job-results/job-2016-04-25T15.51-de316e0/job.log
TESTS      : 24
 (01/24) /usr/share/avocado/tests/passtest.py:PassTest.test;1: WARN
 (02/24) /usr/share/avocado/tests/passtest.py:PassTest.test;2: WARN
 (03/24) /usr/share/avocado/tests/passtest.py:PassTest.test;3: WARN
 (04/24) /usr/share/avocado/tests/passtest.py:PassTest.test;4: WARN
 (05/24) /usr/share/avocado/tests/passtest.py:PassTest.test;5: WARN
 (06/24) /usr/share/avocado/tests/passtest.py:PassTest.test;6: WARN
 (07/24) /usr/share/avocado/tests/passtest.py:PassTest.test;7: WARN
 (08/24) /usr/share/avocado/tests/passtest.py:PassTest.test;8: WARN
 (09/24) /usr/share/avocado/tests/passtest.py:PassTest.test;9: WARN
 (10/24) /usr/share/avocado/tests/passtest.py:PassTest.test;10: WARN
 (11/24) /usr/share/avocado/tests/passtest.py:PassTest.test;11: WARN
 (12/24) /usr/share/avocado/tests/passtest.py:PassTest.test;12: WARN
 (13/24) /usr/share/avocado/tests/failtest.py:FailTest.test;1: FAIL
 (14/24) /usr/share/avocado/tests/failtest.py:FailTest.test;2: FAIL
 (15/24) /usr/share/avocado/tests/failtest.py:FailTest.test;3: FAIL
 (16/24) /usr/share/avocado/tests/failtest.py:FailTest.test;4: FAIL
 (17/24) /usr/share/avocado/tests/failtest.py:FailTest.test;5: FAIL
 (18/24) /usr/share/avocado/tests/failtest.py:FailTest.test;6: FAIL
 (19/24) /usr/share/avocado/tests/failtest.py:FailTest.test;7: FAIL
 (20/24) /usr/share/avocado/tests/failtest.py:FailTest.test;8: FAIL
 (21/24) /usr/share/avocado/tests/failtest.py:FailTest.test;9: FAIL
 (22/24) /usr/share/avocado/tests/failtest.py:FailTest.test;10: FAIL
 (23/24) /usr/share/avocado/tests/failtest.py:FailTest.test;11: FAIL
 (24/24) /usr/share/avocado/tests/failtest.py:FailTest.test;12: FAIL
RESULTS    : PASS 0 | ERROR 0 | FAIL 12 | SKIP 0 | WARN 12 | INTERRUPT 0
JOB HTML   : /home/apahim/avocado/job-results/job-2016-04-25T15.51-de316e0/html/results.html
TIME       : 0.03 s
~~~